### DISCUSSION TITLE
Permission denied

### DISCUSSION THREAD

**Discussion by FCKeijzer91** [OvertureMaps/data#67] [Discussion]
Hi Overture Team, 

I'm struggling with accessing the data. I'm using duckdb and followed the steps of the setup in the README. The Error states 'Permission Denied'. 

I'm not extremely familiar with duckdb and not an expert in SQL (yet). So I'm wondering if I'm doing something wrong or other people experience the same problem. 

Hope you guys can help, would love to play around with the Overture data as in the Webinar hosted by Marc and Jennings.

Kind Regards,
Frans Keijzer

![image](https://github.com/OvertureMaps/data/assets/145649676/07330494-1b50-49cb-848c-120750d573c1)

**Comment by mtravis** [comment]
That looks like a local permissions issue on your machine. Are you able to create files in the folder that you're running DuckDB from?
